iree/compiler/Codegen/Passes.h

```diff
@@ -19,9 +20,15 @@
 namespace mlir {
 namespace iree_compiler {
 
-// Registers all conversion passes in this directory.
+/// Registers all conversion passes in this directory.
 void registerCodegenPasses();
 
+/// Verify that the configuration used for compilation is valid.
+LogicalResult verifyLoweringConfiguration(
```

Comment:
If I read it correctly, this isn't really used and can be deleted?

Reply:
This is the external hook for auto-tuners to verify the configuration.